catroid/src/main/res/values-ar/strings.xml

```diff
@@ -1911,7 +1911,7 @@
     <string name="formula_editor_logic_comparison">عوامل تشغيل المقارنة</string>
     <string name="formula_editor_object_general">الخصائص العامة</string>
     <string name="formula_editor_object_movement">خصائص الحركة</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations